src/storage/consts.rs

```diff
@@ -12,4 +12,5 @@ pub const ID_LENGTH: usize = 20;
 /// Header Constants
 pub mod headers {
     pub const CONTENT_TYPE: &str = "Content-Type";
+    pub const AUTHORIZATION: &str = "Authorization";
```

Comment:
funny enough both of these might not be required.

- [`AUTHORIZATION`](https://docs.rs/hyper/latest/hyper/header/constant.AUTHORIZATION.html)
- [`CONTENT_TYPE`](https://docs.rs/hyper/latest/hyper/header/constant.CONTENT_TYPE.html)